core/vm/gen_structlog.go

```diff
@@ -28,22 +28,24 @@ func (s StructLog) MarshalJSON() ([]byte, error) {
 		Storage       map[common.Hash]common.Hash `json:"-"`
 		Depth         int                         `json:"depth"`
 		RefundCounter uint64                      `json:"refund"`
+		ExtraData     *types.ExtraData            `json:"extraData"`
```

Comment:
Why this file is different from OP?

Reply:
https://github.com/kroma-network/go-ethereum/blob/a4b1216433aae637f79735067834bf1eb13026de/core/vm/logger.go#L86